sources/PTYTextView.m

```diff
@@ -970,6 +978,11 @@ - (void)setNeedsDisplayOnLine:(int)line
     [self setNeedsDisplayOnLine:line inRange:VT100GridRangeMake(0, _dataSource.width)];
 }
 
+- (void)setNeedsDisplayOnCol:(int)col
+{
```

Comment:
Move the `{` up to the end of the preceding line for style consistency

Reply:
Done.